test/remote.ref.access/v1.yml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.1.0
+info:
+  version: 1.0.0
+  title: v1.remote.ref.access.test
+  description: Library test schema
+  license:
+    name: MIT
+
+servers:
+  - url: "http://dev.remote.ref.access.test/v1/"
```

Comment:
NOTE: `.test` TLD is recommended to use in test cases. refs: https://www.rfc-editor.org/rfc/rfc2606#page-2